src/components/Home/Hero/HeroTitleSection.tsx

```diff
@@ -0,0 +1,48 @@
+import cn from 'classnames'
+import React from 'react'
+
+import * as styles from './styles.module.css'
+
+const HeroTitleSection = ({ className }: { className?: string }) => {
+  return (
+    <div className={className}>
+      <div
+        className={cn(
+          'text-center',
+          'px-6 py-10 md:py-24',
+          'flex flex-col gap-6 md:flex-row md:gap-0 items-center',
+          'max-w-screen-lg'
+        )}
+      >
+        <div
+          className={cn(
+            'text-indigo',
+            'flex-1',
+            'md:border-r-2 md:border-solid md:border-r-light',
+            'md:text-left',
+            'lg:pr-16'
+          )}
+        >
+          <h1 className={cn('font-extralight', styles.title)}>
+            <span className="font-medium block sm:inline md:block">
+              Data Version Control
```

Comment:
@volkfox has requested that we add "and much more" to this to pay homage to the brand name while emphasizing that DVC has moved beyond mere data version control.

Something like this?

```suggestion
              Data Version Control<br/><em>&mdash; and much more &mdash;</em>
```

Reply:
https://github.com/iterative/dvc.org/pull/4799